utils/build-script

```diff
@@ -774,6 +778,115 @@ class BuildScriptInvocation(object):
 
         return options
 
+    def compute_product_classes(self):
```

Comment:
If I'm not mistaken, "product_classes" refers to the fact that these are classes (rather than instances) of the products.